code/modules/research/rdconsole.dm

```diff
@@ -372,7 +372,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 
 	linked_destroy.busy = FALSE
 	use_power(DECONSTRUCT_POWER)
-	menu = MENU_MAIN
+	menu = MENU_DESTROY
```

Comment:
wouldn't `menu` already be `MENU_DESTROY` if you just destroyed something? do you need to set it here?

Reply:
yes it would, but I would leave the explicit part of code rather than remove it

Reply:
as there was no reply in the day, I will close this conversation